### ISSUE TITLE
Change "taborder" to "tab order"

### ISSUE THREAD

**Issue by Kenneth-Barber** [tabulartools/pbi-explorer#31]
### Describe the problem

In Report Comparer, in the "Refine comparison" dropdown menu, one of the items is "Hide minor changes (taborder)". This "taborder" should be "tab order".

### PBI Explorer

0.3.1

### Power BI

Irrelevant

### Windows

Irrelevant

**Comment by Kenneth-Barber** [comment]
This issue also appears in the box to the left of the arrow button of the "Refine comparison" dropdown menu when you select any of the "Show" options, plus "Hide minor changes (taborder)", but not "Hide minor changes (positioning)". When you do this, the text will end with "(except TabOrder changes)".

**Comment by DBojsen** [comment]
We'll get that fixed

**Comment by DBojsen** [comment]
This has now been fixed in BETA3.
You can download this version here [https://pbiexplorer.com](https://pbiexplorer.com).

Thank you for reporting.
